### ISSUE TITLE
Unread count in favicon

### ISSUE THREAD

**Issue by Poorchop** [SSilence/selfoss#645]
Since selfoss doesn't have any kind of notification system, it would be cool if the favicon could be updated to reflect the unread message count. For those of us who leave selfoss open in a background tab in the web browser, this would alleviate the need to keep switching to the selfoss tab to check for unread stories. I know that gmail has a system like this in place, so I'm thinking that it should be possible in selfoss too.

**Comment by danieljkemp** [comment]
Currently selfoss does not have any mechanism to update the unread count
(pretty sure on this but someone please correct me if I am wrong). Such a
feature would be nice but you would need to start with a way to get the
page to refresh the unread count after some interval.

**Comment by Poorchop** [comment]
You're right, I forgot that the page doesn't automatically refresh. Even with a cron job updating the feeds, nothing would change unless the page was manually refreshed. I guess this feature would require a pretty big change to the core, maybe using AngularJS to update automatically. That would be a huge improvement to selfoss overall, but that's a much larger issue.

**Comment by jtojnar** [comment]
We now use react so this should be fairly easy to implement with something like https://github.com/oflisback/react-favicon
